File: parts/src/org/lineageos/settings/felica/FelicaSettingsActivity.java
/*
 * Copyright (C) 2026 The LineageOS Project
 *
 * SPDX-License-Identifier: Apache-2.0
 */

package org.lineageos.settings.felica;

import android.os.Bundle;

import com.android.settingslib.collapsingtoolbar.CollapsingToolbarBaseActivity;

public class FelicaSettingsActivity extends CollapsingToolbarBaseActivity {

    private static final String TAG_FELICA = "felica";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        getFragmentManager().beginTransaction().replace(
                com.android.settingslib.collapsingtoolbar.R.id.content_frame,
                new FelicaSettingsFragment(), TAG_FELICA).commit();
    }
}

File: parts/src/org/lineageos/settings/felica/FelicaSettingsActivity.kt
/*
 * Copyright (C) 2026 The LineageOS Project
 * SPDX-License-Identifier: Apache-2.0
 */

package org.lineageos.settings.felica

import android.os.Bundle
import com.android.settingslib.collapsingtoolbar.CollapsingToolbarBaseActivity

class FelicaSettingsActivity : CollapsingToolbarBaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportFragmentManager
            .beginTransaction()
            .replace(
                com.android.settingslib.collapsingtoolbar.R.id.content_frame,
                FelicaSettingsFragment(),
                TAG,
            )
            .commit()
    }

    companion object {
        private const val TAG = "FelicaSettingsActivity"
    }
}
